DEBIAN/postrm

```diff
@@ -11,6 +12,16 @@ case "$1" in
                 db_purge
         fi
         rm -f /usr/sbin/kiosk-browser
+	# In a perfect world, this should also care for lightdm xdm wdm slim lxdm gdm etc...
+	WMS=`dpkg-query -W -f='Package: ${Package}\nProvides: ${Provides}\n'   | grep -B 1 -E "^Provides: .*x-display-manager" | grep Package | cut -d" " -f2`
```

Comment:
The user could have removed a DM which was configured at installtime. Remebering this would not lead to a working DM. 

Will change `` to $()